quam/utils/qua_types.py

```diff
@@ -0,0 +1,45 @@
+__all__ = [
+    "ScalarInt",
+    "ScalarFloat",
+    "ScalarBool",
+    "QuaScalarInt",
+    "QuaScalarFloat",
+    "QuaVariableInt",
+    "QuaVariableFloat",
+    "QuaVariable",
+    "AmpValuesType",
+    "ChirpType",
+    "StreamType",
+    "_PulseAmp",
+]
+
+try:
+    from qm.qua.type_hints import Scalar, QuaScalar, QuaVariable
+
+    ScalarInt = Scalar[int]
+    ScalarFloat = Scalar[float]
+    ScalarBool = Scalar[bool]
+    QuaScalarInt = QuaScalar[int]
+    QuaScalarFloat = QuaScalar[float]
+    QuaVariable = QuaVariable
+    QuaVariableInt = QuaVariable[int]
+    QuaVariableFloat = QuaVariable[float]
+except ImportError:
+    from qm.qua._dsl import QuaNumberType, QuaVariableType, QuaExpressionType
+
+    ScalarInt = QuaNumberType
+    ScalarFloat = QuaNumberType
+    ScalarBool = QuaExpressionType
+    QuaScalarInt = QuaNumberType
+    QuaScalarFloat = QuaNumberType
+    QuaVariable = QuaVariableType
+    QuaVariableInt = QuaVariableType
+    QuaVariableFloat = QuaVariableType
+
+
+from qm.qua._dsl import (
+    _PulseAmp,
+    AmpValuesType,
+    ChirpType,
+    StreamType,
```

Comment:
`_PulseAmp` I know we need to import for now.
`ChirpType` and `StreamType` - try to import from `type_hints`, if it fails, load from _dsl.
Why do we need `AmpValuesType`?

Reply:
Agreed, removed AmpValuesType, changed location of ChirpType and StreamType